docs/source/howto/installation.rst

```diff
@@ -293,68 +293,59 @@ This can be useful to temporarily enable deprecation warnings for a single comma
 
-To isolate the code, make sure to install AiiDA into a virtual environment, e.g., with conda or venv, as described :ref:`here <intro:get_started:setup>`.
-Whenever you activate this particular environment, you will be running the particular version of AiiDA (and all the plugins) that you installed specifically for it.
-
-This is separate from the configuration of AiiDA, which is stored in the configuration directory which is always named ``.aiida`` and by default is stored in the home directory.
-Therefore, the default path of the configuration directory is ``~/.aiida``.
-By default, each AiiDA instance (each installation) will store associated profiles in this folder.
-A best practice is to always separate the profiles together with the code to which they belong.
-The typical approach is to place the configuration folder in the virtual environment itself and have it automatically selected whenever the environment is activated.
-
-The location of the AiiDA configuration folder can be controlled with the ``AIIDA_PATH`` environment variable.
-This allows us to change the configuration folder automatically, by adding the following lines to the activation script of a virtual environment.
-For example, if the path of your virtual environment is ``/home/user/.virtualenvs/aiida``, add the following line:
-
-.. code:: bash
-
-    $ export AIIDA_PATH='/home/user/.virtualenvs/aiida'
-
-Make sure to reactivate the virtual environment, if it was already active, for the changes to take effect.
+An AiiDA instance is defined by its configuration directory, which is always named ``.aiida``.
+It contains the configuration file, which holds all the profile information, daemon log files, and `PID files <https://en.wikipedia.org/wiki/Process_identifier>`_.
 
 .. note::
 
-   For ``conda``, create a directory structure ``etc/conda/activate.d`` in the root folder of your conda environment (e.g. ``/home/user/miniconda/envs/aiida``), and place a file ``aiida-init.sh`` in that folder which exports the ``AIIDA_PATH``.
+    Depending on the storage backend, a profile's data is typically also stored in this directory.
+    However, this is not necessarily always the case, as often the data location can be configured, and some storage backends use a database service that stores the data elsewhere on disk.
 
-You can test that everything works by first echoing the environment variable with ``echo $AIIDA_PATH`` to confirm it prints the correct path.
-Finally, you can check that AiiDA know also properly realizes the new location for the configuration folder by calling ``verdi profile list``.
-This should display the current location of the configuration directory:
+The location of the configuration directory is determined as follows:
 
-.. code:: bash
+1. First, the ``AIIDA_PATH`` environment variables is checked, which can be a colon-separated list of directories.
```

Comment:
Nice, I think this makes sense (and I think mirrors git behaviour?)

Reply:
Kind of. By default git searches the filesystem hierarchy stopping at the first `.git` folder, starting at the CWD. This can be overridden by environment variables, however, there are two different ones: `GIT_DIR` and `GIT_WORK_TREE`. The former can be used to indicate the location of the `.git` folder, and the second the location of the root of the repository that should be considered. This is kind of different to AiiDA where we only really have `GIT_DIR`, which would correspond to `AIIDA_PATH`.

Another difference is that `GIT_DIR` only accepts a single relative or absolute path, not a colon separated list of paths, like `AIIDA_PATH`. Also, `AIIDA_PATH` will accept any path, even if it does not contain a `.aiida` folder, in which case it will simply create it. `GIT_DIR` will error if a path is provided to a `.git` folder that does not exist.

I think the logic of `AIIDA_PATH` is perhaps unnecessarily complex and the simpler behavior of `GIT_DIR` might be sufficient and easier to understand, but I am not sure if we can change this now since someone might rely on it.